public/app/models/query/Condition.js

```diff
@@ -0,0 +1,84 @@
+// Based on http://kindohm.github.io/knockout-query-builder/js/condition.js
+(function () {
+    'use strict';
+
+    define(
+        [
+            'lodash',
+            'knockout'
+        ],
+        function (_, ko) {
+            return function (queryBuilder) {
+                this.selectedField = ko.observable();
+                this.selectedComparator = ko.observable();
+                this.value = ko.observable('');
+
+                this.fields = ko.pureComputed(function () {
+                    return queryBuilder.fields();
+                });
+
+                this.comparators = ko.pureComputed(function () {
```

Comment:
This is hardcoded here, potentially it should probably be brought in from the field definition so that number fields, date fields and text fields can be handled appropriately (e.g. starts-with for text but greater-than for numbers), but in reality all the fields we actually have are text fields, so hardcoding it here is fine.